### PR TITLE
[test] fix flaky test in CI

### PR DESCRIPTION
**Pull request by zhongjiajie**
* 1de3853 : for https://github.com/apache/dolphinscheduler/actions/runs/3065709183/jobs/4952528657
* 0dd4d23 : for https://github.com/apache/dolphinscheduler/actions/runs/3125244808/jobs/5069414026, https://github.com/apache/dolphinscheduler/actions/runs/3060903750/jobs/4949959010, https://github.com/apache/dolphinscheduler/actions/runs/3060903750/jobs/4943713393
* E2E (I find ou all e2e is random failed, it is because `com.github.dockerjava.api.exception.ConflictException: 
Status 409: {"message":"Container 1f0ff7959eb4d1a78a7ba51670c71c438c47d8ec666132fceaa75e94cfa2e2ed is not running"}` or container can not start)
  * UDF
    * https://github.com/apache/dolphinscheduler/actions/runs/3066715967/jobs/4956885089
    * https://github.com/apache/dolphinscheduler/actions/runs/3067976946/jobs/4959380653
  * sqlserver
    * https://github.com/apache/dolphinscheduler/actions/runs/3060875906/jobs/4944080262

### CODE REVIEW

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker [hunk 1]
```diff
@@ -0,0 +1 @@
+mock-maker-inline
```
Comment (zhongjiajie) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker:1]: Add this to mock finial class, according to https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/test/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSenderTest.java [hunk 2]
```diff
@@ -28,14 +33,16 @@
...
 
     @Test
-    public void sendTest() {
+    public void sendTest() throws IOException {
         Map<String, String> paramsMap = new HashMap<>();
-        paramsMap.put(HttpAlertConstants.NAME_URL, "http://www.baidu.com");
+        paramsMap.put(HttpAlertConstants.NAME_URL, "http://www.dolphinscheduler-not-exists-web.com");
         paramsMap.put(HttpAlertConstants.NAME_REQUEST_TYPE, "POST");
         paramsMap.put(HttpAlertConstants.NAME_HEADER_PARAMS, "{\"Content-Type\":\"application/json\"}");
-        paramsMap.put(HttpAlertConstants.NAME_BODY_PARAMS, "{\"number\":\"13457654323\"}");
```
Comment (zhongjiajie) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/test/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSenderTest.java:36]: remove some sensitive data in UT

**Review by kezhenxu94 — DISMISSED** [review]
@EricGao888 @SbloodyS want to double check?

**Review by SbloodyS — APPROVED** [review]
+1

**Review by EricGao888 — APPROVED** [review]
LGTM, thanks

**Comment by zhongjiajie** [comment]
thanks all

**Pull request merged**